packages/toolpad-core/babel.config.js

```diff
@@ -0,0 +1,150 @@
+const path = require('path');
```

Comment:
Got these mostly from Base UI, let me know if there's anything important we should be doing differently as I don't fully understand these build scripts.